estimagic/inference/bootstrap_samples.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+from estimagic.inference.bootstrap_ci import _check_inputs
+
+
+def get_seeds(n_draws=1000):
```

Comment:
Maybe I am missing something, but I would remove this function. I do not see why we need to expose the seeds to the user. All major libraries (e.g. scikit-learn) offer just `seed=None` argument to the main interface and handle the rest internally. Internally, we can sample n seeds with `np.random.randint` as you do if we know how many we need or we could use sequences of seeds with `seed = itertools.count(base_seed); next(seed)` if it is unknown.

Reply:
I think here the reasoning was that one can easily calculate bootstrap estimates with few seeds and then add other seeds later. But this can also be done with the solution you propose.

Reply:
Will change in #202